extender/kaniko/dockerfile_applier.go

```diff
@@ -104,9 +87,9 @@ func (a *DockerfileApplier) ApplyRun(dockerfiles []extender.Dockerfile, baseImag
 			CustomPlatform:  platforms.DefaultString(),
 
 			Cache:     true,
-			CacheRepo: cacheRef,
+			CacheRepo: a.cacheImageRef,
 			CacheOptions: config.CacheOptions{
-				CacheTTL: 14 * (24 * time.Hour),
+				CacheTTL: 14 * (24 * time.Hour), // TODO: should this be configurable?
```

Comment:
Probably? Platforms can already dump the cache image itself, but this configuration seems useful to me.